hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1034,6 +1040,9 @@
               phase.
             - After sysrst reset is generated by forcing, read the reset cause register in rstmgr to confirm that the SW is now in
               the sysrst reset phase.
+            - Generate sysrst by driving input PAD.
+            - After reset, read the reset cause register in rstmgr to confirm that the SW is now in
+              the sysrst reset phase.
             - Program the AON timer wdog to 'bark' after some time.
```

Comment:
As mentioned above, this comment is regarding chip_sw_pwrmgr_sysrst_ctrl_reset rather than this PR. To answer this question, chip_sw_pwrmgr_sysrst_ctrl_reset is to generate a sysrst_ctrl HW reset after a sleep. Watchdog reset is placed to make two sequential reset events starting from sysrst_ctrl HW reset. I will add more texts to be more clear in another PR. Thanks!